docs/VLSI/Tutorial.rst

```diff
@@ -78,17 +78,13 @@ Pull the Hammer environment into the shell:
 
 Building the Design
 --------------------
-To elaborate the ``Sha3RocketConfig`` (Rocket Chip w/ the accelerator) and set up all prerequisites for the build system to push just the accelerator + hard macro through the flow:
+To elaborate the ``TinyRocketConfig`` and set up all prerequisites for the build system to push just the accelerator + hard macro through the flow:
```

Comment:
I'm confused - if we switched from Sha3RocketConfig to TinyRocketConfig, is there still an accelerator and hard macro?